Cargo.toml

```diff
@@ -13,13 +13,13 @@ categories = ["data-structures", "cryptography::cryptocurrencies"]
 [dependencies]
 educe = "0.6.0"
 ethereum_hashing = "0.7.0"
-ethereum_ssz = "0.8.0"
-ethereum_ssz_derive = "0.8.0"
+ethereum_ssz = "0.8"
+ethereum_ssz_derive = "0.8"
```

Comment:
These changes are cosmetic (0.8 and 0.8.0 are equivalent), but I'll just merge as is

Reply:
Ah of course, I frequently make this mistake.